### PR TITLE
mention force sync for Travis CI

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

**Comment by ocefpaf** [comment]
Awesome! Thanks!!

File: docs/_sources/ciservices.rst.txt [hunk 1]
```diff
@@ -8,11 +8,18 @@ CircleCI (Linux)
 Circle CI is a container-based CI service that conda-forge uses to build
 linux packages.
 
+
+Travis CI (OSX)
+------------------------------
+Travis CI is used to build packages for OSX. After merging a staged-recipes pull request, it might be necessary to
+force sync your repositories in Travis CI to see the reload and cancel buttons. To do this please visit "https://travis-ci.org/profile/<github_user_name>" and click "Sync accounts".
```
Comment (isuruf) [inline docs/_sources/ciservices.rst.txt]: You can remove the `<github_user_name>` part and make this a link

File: docs/_sources/ciservices.rst.txt [hunk 2]
```diff
@@ -8,11 +8,18 @@ CircleCI (Linux)
...
+
+
```
Comment (ocefpaf) [inline docs/_sources/ciservices.rst.txt]: Can you move this up `CircleCI (Linux)` or all the way down. This ended up in the middle :smile:  (` Enabling Circle on your Fork` is a subpart of `CircleCI (Linux)`).

File: docs/_sources/ciservices.rst.txt [hunk 3]
```diff
@@ -8,11 +8,18 @@ CircleCI (Linux)
 Circle CI is a container-based CI service that conda-forge uses to build
 linux packages.
 
+
+Travis CI (OSX)
```
Comment (ocefpaf) [inline docs/_sources/ciservices.rst.txt]: This is minor but it should be `OS X` instead of `OSX`.

**Comment by jakirkham** [comment]
Umm...I think you want the file in `src/`. This is an autogenerated file.

**Comment by bgruening** [comment]
@jakirkham mh, good point. md files are autogenerated by md files?
I created #381

**Comment by jakirkham** [comment]
Sorry about that @bgruening. Thanks for correcting it.

**Comment by bgruening** [comment]
@jakirkham no problem, learning as I go! :)

**Pull request closed without merging**